msticpy/common/wsconfig.py

```diff
@@ -92,7 +93,13 @@ def __init__(self, config_file: Optional[str] = None, workspace: str = None):
                     "\n".join(_NO_CONFIG_WARN).format(config_file=config_file)
                 )
         self._config_file = config_file
-        self._config.update(self._read_config_values(config_file))
+        config = self._read_config_values(config_file)
+        if config is not None:
+            self._config.update(config)
+        else:
+            os.environ["MSTICPYCONFIG"] = config_file
```

Comment:
I wonder if this might cause some issues. E.g. if I already have a MSTICPYCONFIG env var but wanted to force use of a local msticpyconfig just for workspace settings. It's probably OK to leave it as you have it. It will be a pretty rare use case anyway and it's probably a nightmare to support simultaneous use of multiple msticpyconfig.yamls